src/components/LikeThought.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import './Thoughts.css'
+import { GetThoughts } from './Thoughts'
+
+export const LikeThoughts = ({ index }) => {
```

Comment:
Maybe this prop passed to LikeThoughts is named index is a bit misleading. Consider others, like "thought" might be better I think.